app/Http/Resources/TrustedUserResource.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+/**
+ * @OA\Schema(
+ *     title="TrustedUser",
+ *     @OA\Property(property="user", ref="#/components/schemas/LightUser"),
+ *     @OA\Property(property="expires_at", type="string", format="date-time", example="2024-07-28T00:00:00Z")
+ * )
+ */
+class TrustedUserResource extends JsonResource
+{
+    public function toArray($request): array {
+        return [
+            'user'       => new LightUserResource($this->trusted),
+            'expires_at' => $this->expires_at?->toIso8601String()
```

Comment:
maybe `created_at`?

Reply:
(Also: `expiresAt` and `createdAt`. We use camel case in the api. :D)

Reply:
I don't see a reason for adding `created_at` - it's just more data to sent?

Reply:
imho it could be beneficial for the enduser to know how long ago they've added a user. But maybe that's just me.